### ISSUE TITLE
built-in `makeheader` and `stripcomments` not working at all.

### ISSUE THREAD

**Issue by 7dog123** [Olde-Skuul/burgerlib#11]
I'm getting errors: `generated error [Errno 13] Permission denied` when using the built-in makeheader and stripcomments. Then, when I make them executable, I get `generated error [Errno 8] Exec format error`. Both files are only 1 KB on the tools linux, macosx, and windows folders. Inside both files, the contents are:

makeheader:

```
version https://git-lfs.github.com/spec/v1
oid sha256:90cbfd1fde371b78e3a4f04a42a40ee9cbfcafded7e873d13d52382df5fb0ea0
size 275840
```

stripcomments:

```
version https://git-lfs.github.com/spec/v1
oid sha256:059aac7bfd2f8d544dfbbaae6cd6f6c6c9e369d33068850c15ac6cb5eed1801c
size 270688
```

**Comment by burgerbecky** [comment]
Do you have git-lfs installed? Those files are stored as git-lfs entries.

**Comment by 7dog123** [comment]
Yeah, I kinda learned that the hard way. Also, there are a couple of files in docs/manuals—M68000PRM.pdf and MPEG-2_Audio_IS.pdf—that couldn't be found when building with the Doxyfile. If you check [line 1504](https://github.com/Olde-Skuul/burgerlib/blob/774a8e5073b61d99bb8f14dfba0e8b3068b03c47/docs/Doxyfile#L1504) and [line 1511](https://github.com/Olde-Skuul/burgerlib/blob/774a8e5073b61d99bb8f14dfba0e8b3068b03c47/docs/Doxyfile#L1511) in the Doxyfile, the filenames are in lowercase, while the actual filenames in the docs/manuals folder use a mix of uppercase and lowercase letters.

**Comment by burgerbecky** [comment]
Ah, I'll fix that. Thank you for pointing that out.
